### ISSUE TITLE
Possibly treating http and https as seperate origins

### ISSUE THREAD

**Issue by MarcusJT** [EFForg/privacybadgerchrome#209]
PBC breaks the [Hootsuite Hootlet](https://chrome.google.com/webstore/detail/hootsuite-hootlet/bjgfdlplhmndoonmofmflcbiohgbkifn) and [Save to Pocket](https://chrome.google.com/webstore/detail/save-to-pocket/niloccemoadcdkdjlinkgdfekeahmflj) extensions - after much fiddling (thanks to no hints in Chrome console that PBC was doing the blocking) I found that disabling PBC makes them work again.

The Hootsuite extension can also be made to work by manually setting the hootsuite.com domain back to green in PBC settings but oddly there is no equivalent domain listed for the getpocket.com requests which are being blocked, and in any case it surely shouldn't be necessary to manually unblock these popular extensions.

**Comment by tsibley** [comment]
@artlogic I just came across this issue trying to figure out the same thing as you. I realized I could provoke the Badger artifically and then change the blocking settings, so I made a little tool to do so: https://tsibley.net/provoke-the-privacy-badger/.  I hope it helps!

**Comment by artlogic** [comment]
@tsibley very nice! Works like a charm. I had been doing something similar to this, but using the browser's dev tools. Your solution is much more pleasant.

**Comment by MarcusJT** [comment]
Excellent, thanks @tsibley!

I hope that in the long run PB will implement this (or something similar) itself... @cooperq ?

**Comment by tsibley** [comment]
Yeah, I do hope PB will implement a way to change privacy settings without being on the page were they were triggered.

**Comment by cooperq** [comment]
There is a pull request in the works right now to have an options page where you can change settings for all of the domains privacy badger knows about. It should be released soon.

**Comment by tsibley** [comment]
@cooperq Yay!

**Comment by timmib** [comment]
As of today I am still having instapaper.com bookmarklet interrupted even though the slider is green.

**Comment by tsibley** [comment]
@timmib It looks like Privacy Badger may have started distinguishing between http:// and https:// schemes in its whitelist.  Until that pull request that @cooperq was talking about lands, try using https://tsibley.net/provoke-the-privacy-badger/ with instapaper.com (again, it's updated).  Doing so just solved the problem (again!) for me.

**Comment by cooperq** [comment]
@tsibley privacy badger should not be distinguishing between http and https in the cookie block list. The pull request has landed and will be in the next release. 
If privacy badger is treating http and https as separate domains that is a bug, what leads you to believe it is?

**Comment by tsibley** [comment]
@cooperq I'd successfully whitelisted a domain using a triggered HTTP request and it was working great.  Then when I last updated Privacy Badger (in Firefox, on 1 April), that whitelisting stopped working.  As far as I could tell, the domain was still considered whitelisted under HTTP, but not HTTPS.  Once I toggled the slider to green after triggering an HTTPS request to the domain, the whitelisting was restored.

This wasn't a controlled test case, just what I seemed to be observing.  It could, of course, be for another reason entirely.
